src/mutations/user-interaction.mutation.ts

```diff
@@ -0,0 +1,19 @@
+import { gql } from '@apollo/client/core';
+
+export const USER_LIKE_ENTITY = gql`
+    mutation($input: UserInteractionInput!) {
+        userLikeEntity(input: $input)
+    }
+`;
+
+export const USER_UN_lIKE_ENTITY = gql`
+    mutation($input: UserInteractionInput!) {
+        userUnLikeEntity(input: $input)
+    }
+`;
+
+export const  USER_DIS_LIKE_ENTITY = gql`
```

Comment:
Question: what's the difference in user interaction between UnLike and DisLike?

Reply:
unLike is remove previous like , Dislike is another interaction, is as "I don't like this post"